flag/service/kubernetes/kubernetes.go

```diff
@@ -8,8 +8,9 @@ import (
 // Kubernetes is a data structure to hold Kubernetes specific command line
 // configuration flags.
 type Kubernetes struct {
-	Address   string
-	InCluster string
-	TLS       tls.TLS
-	Watch     watch.Watch
+	Address    string
+	InCluster  string
+	KubeConfig string
+	TLS        tls.TLS
+	Watch      watch.Watch
```

Comment:
What do you think about having the common flags in `operatorkit` and using the structures in the operators' flag package by importing? We could then be consistent via vendor updates.

Reply:
Yes it makes sense to move the flags to operatorkit. I'd like to add the kubeconfig support to all operators so I can use the vendored flags then.

Reply:
Done. PTAL.